### PR TITLE
Personas 

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/platform/corda/5.0/introduction/personas.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* work as part of a team to build an enterprise production-grade distributed application.
+* understand the software development lifecycle (SDLC) of a distributed application.
+* implement a method in the API to make something simpler.
+* discover best practice design tips for distributed applications.
+
+To get started developing CorDapps, see the [Developing Applications]({{< relref "../developing-applications/_index.md">}}) section.
+
+## Corda Administrator
+
+A Corda administrator is responsible for:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0/introduction/personas.md:49]:
```suggestion
A Corda Administrator is responsible for:
```
Not sure about capitalization strategy here. You capitalize "Architect" and both words in "CorDapp Developer".

File: content/en/platform/corda/5.0/introduction/personas.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* integrating Corda into Continuous Integration and Delivery CI/CD pipelines.
+* understanding how Corda manages errors to deliver continuous execution.
+* the disaster recovery story for Corda.
+* monitoring the health metrics of Corda.
+
+To learn more, see the [Operating Corda Clusters]({{< relref "../deploying-operating/_index.md">}}) section.
+
+## Network Operator
+
+The network operator is concerned with:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0/introduction/personas.md:61]:
```suggestion
The Network Operator is concerned with: 
```
See my previous comment about capitalization.

File: content/en/platform/corda/5.0/introduction/personas.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Network Operator
+
+The network operator is concerned with: 
+* how Corda delivers on the _permissioned_ aspect of a private permissioned DLT platform.
+* service levels for participants on the business network.
+* defining permissions for actions on the business network.
+* “know your customer” (KYC) regulations.
+
+The network operator:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0/introduction/personas.md:67]:
```suggestion
The Network Operator:
```

**Review by yarivus — COMMENTED** [review]
thx @AnnaK-R3 and @nadinequinr3

**Pull request merged**